WcaOnRails/config/locales/en.yml

```diff
@@ -1021,6 +1022,31 @@ en:
         content:
           #context: Yup, nothing to translate here! Just hit "Use the original version"
           '1': "%{current_user_claim}"
+      '9':
+        title: "I am registered for a competition and want to sign up for an additional event, what should I do? Can I bring guests?"
+        #context: Answer to "I want to sign up or an additional event ...?"
+        content:
+          '1': "Please contact the organizer. You can find the contact information on the competition website. If you are unsure where to find it, check the <a href='/competitions'>list of competitions</a>."
```

Comment:
Should we be using `%{comp_search_form}` here? Or maybe link to https://www.worldcubeassociation.org/faq#how-can-i-register-for-a-competition-who-can-i-refer-to-if-i-have-problems-registering-for-a-competition?